mmdet/datasets/coco.py

```diff
@@ -383,26 +383,28 @@ def format_results(self, results, jsonfile_prefix=None, **kwargs):
         result_files = self.results2json(results, jsonfile_prefix)
         return result_files, tmp_dir
 
-    def evaluate(self,
-                 results,
-                 metric='bbox',
-                 logger=None,
-                 jsonfile_prefix=None,
-                 classwise=False,
-                 proposal_nums=(100, 300, 1000),
-                 iou_thrs=None,
-                 metric_items=None):
-        """Evaluation in COCO protocol.
+    def evaluate_det_segm(self,
```

Comment:
Have you checked that this is consistent with the original implementation?
We should be very cautious about any changes to the metric.